l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -197,16 +202,18 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
     }
 
     function stmAssetIdFromChainId(uint256 _chainId) public view override returns (bytes32) {
+        // TODO: should we generate the assetid even for chains that don't have state transition manager set?
         return stmAssetId(stateTransitionManager[_chainId]);
     }
 
     function stmAssetId(address _stmAddress) public view override returns (bytes32) {
+        // TODO: check that the STM is registered.
```

Comment:
could you please add it? it is just one require

Reply:
done